AnnoyancesFilter/Popups/sections/popups_specific.txt

```diff
@@ -7,6 +7,7 @@
 !
 ! SECTION: Popups - Regular rules
 !
+/75027.f98ffb777033e1b.js$domain=coinmarketcap.com
```

Comment:
I don't see this resource on my end. I can't reproduce the popup either.  @AdguardTeam/filters-maintainers could you please take a look?

Reply:
I just loaded the website, and the file seems to be gone now, along with the popup. I will try loading it in a few different browsers and on a different IP. If it still does not load, I will close this.